packages/workspace/src/utils/default-base.ts

```diff
@@ -0,0 +1,19 @@
+import { execSync } from 'child_process';
+
+export function deduceDefaultBase(fallback: string = 'main'): string {
```

Comment:
I don't think I've seen you pass `fallback` here :eyes:

Reply:
I can remove the parameterization here, if you think its better without it. I pass it in a few tests, but otherwise yeah its not used currently.